<launch>
  <node pkg="map_provider" exec="map_provider" name="map_provider" output="screen">
    <remap from="output/pointcloud_map" to="/map/pointcloud_map/partial"/>
    <remap from="pcd_loader_service" to="/map/load_pcd_partially"/>
    <remap from="pcd_provider_service" to="/map_provider/provide_map"/>

    <param name="pointcloud_map_radius" value="200.0"/>
    <param name="update_threshold_distance" value="50.0"/>
  </node>
</launch>
